### PR TITLE
Additional profiles

### PR DESCRIPTION
**Pull request by pieter-edelman-nictiz**

### CODE REVIEW

File: Resources/lu-LaboratoryTestResult-DiagnosticReport.xml [hunk 1]
```diff
@@ -0,0 +1,225 @@
...
+        <targetProfile value="http://nictiz.nl/fhir/StructureDefinition/nl-core-Patient" />
+      </type>
+      <mustSupport value="true" />
+      <mapping>
+        <identity value="lu-dataset-2" />
+        <map value="lu-concept-v2-4303" />
+        <comment value="Patient" />
+      </mapping>
+    </element>
+    <element id="DiagnosticReport.issued">
```
Comment (pieter-edelman-nictiz) [inline Resources/lu-LaboratoryTestResult-DiagnosticReport.xml]: Why is this labeled mustSupport? There's no mapping to the functional model.

Last time, @njnikodym said:
>When searching for the "newest" result is technically a relevantie element. Currently it is not in the dataset but I'm discussing the importance of it with IA

However, I think this is the wrong way around. If this information is required for the use case, it should be defined on a functional level first.

File: Resources/lu-LaboratoryTestResult-DiagnosticReport.xml [hunk 2]
```diff
@@ -0,0 +1,225 @@
...
+  <contact>
+    <name value="Nictiz" />
+    <telecom>
+      <system value="url" />
+      <value value="https://www.nictiz.nl" />
+      <use value="work" />
+    </telecom>
+  </contact>
+  <description value="A Diagnostic Report for Laboratory exchange based on the Dutch Health and Care Information models (Dutch: Zorginformatiebouwsteen or ZIB)  version 4.6, release 2020." />
+  <purpose value="# Purpose of HCIM\n\nLaboratory tests are done for the purpose of diagnosing and preventing disease and follow-up on the effects of treatment." />
```
Comment (pieter-edelman-nictiz) [inline Resources/lu-LaboratoryTestResult-DiagnosticReport.xml]: I would expect an explanation for the existence of this profile (so not what it's for, but why it's needed).

File: Resources/lu-LaboratoryTestResult-DiagnosticReport.xml [hunk 3]
```diff
@@ -0,0 +1,225 @@
...
+  <kind value="resource" />
+  <abstract value="true" />
+  <type value="DiagnosticReport" />
+  <baseDefinition value="http://hl7.org/fhir/StructureDefinition/DiagnosticReport" />
+  <derivation value="constraint" />
+  <differential>
+    <element id="DiagnosticReport">
+      <path value="DiagnosticReport" />
+      <short value="LaboratoryTestResult" />
+      <definition value="The findings and interpretation of diagnostic tests performed on patients, groups of patients, devices, and locations, and/or specimens derived from these. The report includes clinical context such as requesting and provider information, and some mix of atomic results, images, textual and coded interpretations, and formatted representation of diagnostic reports." />
```
Comment (pieter-edelman-nictiz) [inline Resources/lu-LaboratoryTestResult-DiagnosticReport.xml]: I would expect guidance here on how this profile relates to the nl-core profiles. I'm not actually sure what should be said here.

File: Resources/lu-LaboratoryTestResult-DiagnosticReport.xml [hunk 4]
```diff
@@ -0,0 +1,225 @@
...
+  <publisher value="Nictiz" />
+  <contact>
+    <name value="Nictiz" />
+    <telecom>
+      <system value="url" />
+      <value value="https://www.nictiz.nl" />
+      <use value="work" />
+    </telecom>
+  </contact>
+  <description value="A Diagnostic Report for Laboratory exchange based on the Dutch Health and Care Information models (Dutch: Zorginformatiebouwsteen or ZIB)  version 4.6, release 2020." />
```
Comment (pieter-edelman-nictiz) [inline Resources/lu-LaboratoryTestResult-DiagnosticReport.xml]: Why not use the blurb from the zib here?
>A laboratory result describes the result of a laboratory analysis. These are specimen-oriented tests as performed in laboratories such as Clinical Chemistry, Serology, Microbiology, etc. In addition to the results of tests with a singular result, this concept can also contain the results of more complex tests with multiple results or a ‘panel’.

File: Resources/lu-LaboratoryTestResult-DiagnosticReport.xml (hunk 1 above)
Comment (njnikodym) [reply Resources/lu-LaboratoryTestResult-DiagnosticReport.xml]: For now, I will take this out of the profile as it is still a task to discuss with the IA to incorporate this on a functional level

File: Resources/lu-LaboratoryTestResult-DiagnosticReport.xml (hunk 3 above)
Comment (njnikodym) [reply Resources/lu-LaboratoryTestResult-DiagnosticReport.xml]: As there is currently no nl-core profile for DiagnosticReport, I could mention something along those lines in here

**Review by njnikodym — APPROVED** [review]

**Pull request merged**